scipy/stats/tests/test_distributions.py

```diff
@@ -5204,3 +5204,32 @@ def _rvs(self):
 
     with assert_warns(np.VisibleDeprecationWarning):
         rvs_no_size.rvs()
+
+
+@pytest.mark.parametrize(
```

Comment:
Can this be shared with the list added in gh-13283, or is it substantially different?

Reply:
It won't be very different for the discrete distributions. But will have to add invalid parameters for the continuous distributions. We can add these lists of invalid parameters in the `_dist_params.py` file later, in a separate PR. What do you say?

Reply:
I see. OK.